### PR TITLE
Notes/GitHub and markdown

### PR DESCRIPTION
**Pull request by sirtobiwan**
pls check Readme 3

### CODE REVIEW

**Comment by Irazhap** [comment]
Nice Work!

**Comment by peterpane1990** [comment]
super duper

**Review by Irazhap — COMMENTED** [review]
Awesome!

**Review by Irazhap — CHANGES_REQUESTED** [review]
Awesome!

**Review by peterpane1990 — APPROVED** [review]
tip top

File: README3.md [hunk 1]
```diff
@@ -0,0 +1 @@
+Neuer Versuch Branch mit README3
```
Comment (Irazhap) [inline README3.md:1]: Keine Ahnung!

**Review by Irazhap — APPROVED** [review]

**Pull request merged**